packages/backend/src/ee/clients/Slack/SlackBot.ts

```diff
@@ -276,10 +277,12 @@ export class CommercialSlackBot extends SlackBot {
         });
     }
 
+    // WARNING: Needs - channels:history scope for all slack apps
```

Comment:
should it be added here? 
https://github.com/lightdash/lightdash/blob/f71e79f5ae58646605e9a13a8935df885b6f8f4d/packages/common/src/types/slackSettings.ts#L15-L27

Also how does adding scope work? they'd have to re-install the slack app right? Which means they have to re-configure all agents?

Reply:
Added consent screen when summoned: 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/DJhzIQbRJqTJiKN3mUjT/5d97a904-5f2d-4e92-851b-b2079416c117.png)


this info (consent) is stored on a separate table - we can chat about this implementation